src/main/java/step2/domain/WinningPrice.java

```diff
@@ -0,0 +1,31 @@
+package step2.domain;
+
+import java.util.stream.Stream;
+
+public enum WinningPrice {
+
+  FIRST(2000000000, 6),
```

Comment:
```suggestion
  FIRST(2_000_000_000, 6),
```
위처럼 사용해서 가독성을 향상 시킬 수 있어요